test/execution/atomics_test.cpp

```diff
@@ -11,13 +11,27 @@
 #include "execution/vm/module.h"
 #include "execution/vm/vm_defs.h"
 #include "spdlog/fmt/fmt.h"
+#include "test_util/fs_util.h"
 #include "test_util/multithread_test_util.h"
 
 namespace noisepage::execution::test {
 
 class AtomicsTest : public TplTest {
  public:
-  AtomicsTest() : region_("atomics_test") { vm::LLVMEngine::Initialize(); }
+  static void SetUpTestSuite() {
+    // TODO(Kyle): We could just as easily locate the project root path here and then
+    // manually concatenate the path at which we expect the bytecode handlers file will
+    // be located, but this approach feels more robust, at the cost of some additional search.
+    bytecode_handlers_path =
```

Comment:
Seems like logic that you might want to extract into `TplTest` or even some new `TplBytecodeTest`.

Reply:
Setup + TearDown + getter in particular

Reply:
Good idea. Pulled it out into a new `CompiledTplTest` because `TplTest` itself is relatively general purpose (used extensively, for many non-compiled tests).